testing/test_mark.py

```diff
@@ -477,6 +516,19 @@ def test_some(request):
         reprec = testdir.inline_run("-m", "mark1")
         reprec.assertoutcome(passed=1)
 
+    def assert_markers(self, items, **expected):
```

Comment:
if other tests need the functioanlity it could go to testdir. can't judge right now.